man/check-os-release.py

```diff
@@ -0,0 +1,28 @@
+#!/bin/python
```

Comment:
nit: `/usr/bin/`? Give how we try to push for usr-merge :-)

Reply:
The `/bin` symlink will always stay on usr-merged systems, so there isn't any reason to prefer /usr/bin/python of /bin/python. But yeah, /usr/bin/python is what we use internally, so I'll change this too.